feat(macros): map Q and @x to repeat visually (#26495)
neovim/neovim: runtime/doc/visual.txt

be applied up to the rightmost column of the longest line.  Any count passed
to the `.` command is not used.

> With registers, apply the desired register to each line selected in Visual mode.
> For example:
> 
> 	123(hello)321
> 	456(world)654
> 	456(NOT THIS)654
> 
> Having a register `@x` containing the following feed `yi(VP`, visually selecting the
> first two lines and calling `@x`, the output will be as follows:
> 
> 	hello
> 	world
> 	456(NOT THIS)654

Reviewer: This may be a bit nitty of me, but I feel like this section ought to be clear that this is describing a mapping.
I also feel the wording can be improved; e.g: "calling" is a term usually reserved for functions.

I made a quick attempt to improve this below: (I'm still slightly unhappy with it; it's a tad verbose?)

```suggestion
Visual mode |default-mappings| "@" and "Q" can repeat commands in a register
for all selected lines. See |v_@-default| and |v_Q-default| for details. For
example, given the text:

	123(hello)321
	456(world)654
	456(NOT THIS)654

With register "x" containing the commands "yi(VP", Visually selecting the
first two lines and typing "@x" produces:

	hello
	world
	456(NOT THIS)654
```

be applied up to the rightmost column of the longest line.  Any count passed
to the `.` command is not used.

> With registers, apply the desired register to each line selected in visual mode.

Reviewer 2: ```suggestion
With registers, apply the desired register to each line selected in Visual mode.
```

For example:

	123(hello)321
	456(world)654
	456(NOT THIS)654

> Having a register `@x` containing the following feed `yi(VP`.
> By visually selecting the first two lines and calling `@x`,
> the output will be as follows:

Reviewer 2: ```suggestion
Having a register `@x` containing the content `yi(VP`, visually selecting the
first two lines and typing `@x`, the output will be as follows:
```
